web/src/settings_users.js

```diff
@@ -467,6 +468,49 @@ export function set_up_humans() {
     start_data_load();
     section.active.handle_events();
     section.deactivated.handle_events();
+
+    const userToggler = components.toggle({
+        child_wants_focus: true,
+        values: [
+            {label: $t({defaultMessage: "Users"}), key: "user-list-admin"},
+            {label: $t({defaultMessage: "Deactivated users"}), key: "deactivated-users-admin"},
+            {label: $t({defaultMessage: "Invitations"}), key: "invites-list-admin"},
+        ],
+        callback(_name, key) {
+            if (key !== "user-list-admin") {
+                $("#admin-user-list .user-section-container").hide();
+            } else {
+                $("#admin-user-list .user-section-container").show();
+            }
+
+            if (key === "deactivated-users-admin") {
+                allow_sorting_deactivated_users_list_by_email();
+            }
+            activate_user_section(key);
+        },
+    });
+    $("#admin-user-list .tab-container").append(userToggler.get());
```

Comment:
There is no code to load the invitations data, so no invitations are shown on toggling to the "invitations" tab.

Reply:
I have updated the changes to load the invitations data.
Hope that works.